packages/storybook/stories/components/modal/modal.stories.ts

```diff
@@ -0,0 +1,240 @@
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { defineCustomElement } from '@ovhcloud/ods-components/dist/components/ods-modal';
+import { ODS_MODAL_COLOR, ODS_MODAL_COLORS } from '@ovhcloud/ods-components';
+import { html } from 'lit-html';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+import { CONTROL_CATEGORY, orderControls } from '../../control';
+
+defineCustomElement();
+
+const meta: Meta = {
+  title: 'ODS Components/User Feedback/Modal',
+  component: 'ods-modal',
+};
+
+export default meta;
+
+export const Demo: StoryObj = {
+  render: (args) => html`
+<ods-modal class="modal-demo" is-open="${args.isOpen}" is-dismissible="${args.isDismissible}" color="${args.color}">
+  ${unsafeHTML(args.content)}
+  ${unsafeHTML(args.actions)}
+</ods-modal>
+<style>
+  .modal-demo .my-text::part(text) {
```

Comment:
weird to have custom style (that user can't interact with) on a control that is customisable